app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -46,21 +47,16 @@ def security_event_tokens
       return {} unless timestamp
```

Comment:
The return type of this is now an array so lets make that the case for this.
```suggestion
      return [] unless timestamp
```

I'd recommend some YARD documentation for future us as well (but couldn't do via suggestion)
```ruby
    # @return [Array<String>] JWE strings
    def security_event_tokens
```

Reply:
Done!